tianshou/trainer/__init__.py

```diff
@@ -5,15 +5,15 @@
 from tianshou.trainer.utils import test_episode, gather_info
 from tianshou.trainer.onpolicy import onpolicy_trainer, onpolicy_trainer_generator
 from tianshou.trainer.offpolicy import offpolicy_trainer, offpolicy_trainer_generator
-from tianshou.trainer.offline import offline_trainer, offline_trainer_generator
+from tianshou.trainer.offline import offline_trainer, offline_trainer_iter
 
 __all__ = [
     "offpolicy_trainer",
     "offpolicy_trainer_generator",
     "onpolicy_trainer",
     "onpolicy_trainer_generator",
     "offline_trainer",
-    "offline_trainer_generator",
+    "offline_trainer_iter",
```

Comment:
Maybe we can directly expose `OfflineTrainer`? Though it may be a little bit confusing... and also requires some documentation update.

Reply:
Yes I got an error saying so. What do you propose?